scripts/modules/task-manager/update-tasks.js

```diff
@@ -414,7 +442,8 @@ async function updateTasks(
 				aiServiceResponse.mainResult,
 				tasksToUpdate.length,
 				logFn,
-				isMCP
+				isMCP,
+				promptVariant // Pass the promptVariant to handle gemini-cli
```

Comment:
now we're adding more complexity to the code instead of resolving the core issue